src/scripts/syncBreaches.js

```diff
@@ -11,7 +11,83 @@
 
 import { req, formatDataClassesArray } from '../utils/hibp.js'
 import { getAllBreaches, upsertBreaches } from '../db/tables/breaches.js'
+import { mkdir, readdir } from "node:fs/promises";
+import { resolve as pathResolve } from "node:path";
+import { finished } from 'stream/promises';
+import fs from "fs";
+import { Readable } from 'stream';
```

Comment:
Same here re: `node:stream`
```suggestion
import { Readable } from 'node:stream';
```

Possibly also sort so the fs* and stream* dependencies are closer.
I guess we could also possibly group these into a single, albeit confusing, import like:

```js
import { Readable, promises: stream  } from "node:stream";
```
Where now Readable is, well, whatever that is, and `stream` alias is the async promise-API methods. Although not sure that's any better/clearer, and now I'm just over-nit-ing.